src/loader-components/PropsTable.module.scss

```diff
@@ -14,4 +15,13 @@
     white-space: normal;
     word-wrap: break-word;
   }
+
+  tr {
+    background-color: var(--terra-dev-site-props-table-background-color, #fff);
+    border-top: 1px solid #c6cbd1;
+  }
+
+  tr:nth-child(2n) {
+    background-color: var(--terra-dev-site-props-table-alt-background-color, #f6f8fa);
```

Comment:
I'm not sure how best to name this, I believe this is alternating the rows every other. I'd at a minimum scope to tr:

`--terra-dev-site-props-table-tr-alt-background-color`